packages/paste-core/components/tabs/stories/index.stories.tsx

```diff
@@ -76,59 +186,63 @@ const useButtonClickTabState = (): TabStateReturn => {
   };
 };
 
-export const StateHookExample: React.FC = () => {
+export const StateHookTabs: React.FC = () => {
   const {...tab} = useButtonClickTabState();
   return (
-    <>
-      <Paragraph>These tabs use the state prop to allow a button in the tab content to move between tabs.</Paragraph>
-      <Tabs state={tab}>
-        <TabList aria-label="My tabs">
-          <Tab>Tab 1</Tab>
-          <Tab>Tab 2</Tab>
-        </TabList>
-        <TabPanels>
-          <TabPanel>
-            <Paragraph>Tab 1</Paragraph>
-            <Button variant="primary" onClick={() => tab.select('state-hook-tab-example-2')}>
-              Go to tab 2
-            </Button>
-          </TabPanel>
-          <TabPanel>
-            <Paragraph>Tab 2</Paragraph>
-            <Button variant="primary" onClick={() => tab.select('state-hook-tab-example-1')}>
-              Go back to tab 1
-            </Button>
-          </TabPanel>
-        </TabPanels>
-      </Tabs>
-    </>
+    <Tabs state={tab}>
+      <TabList aria-label="Tabs to test programmatic state handling">
+        <Tab>Tab 1</Tab>
+        <Tab>Tab 2</Tab>
+      </TabList>
+      <TabPanels>
+        <TabPanel>
+          <Paragraph>Tab 1</Paragraph>
+          <Button variant="primary" onClick={() => tab.select('state-hook-tab-example-2')}>
+            Go to tab 2
+          </Button>
+        </TabPanel>
+        <TabPanel>
+          <Paragraph>Tab 2</Paragraph>
+          <Button variant="primary" onClick={() => tab.select('state-hook-tab-example-1')}>
+            Go back to tab 1
+          </Button>
+        </TabPanel>
+      </TabPanels>
+    </Tabs>
   );
 };
-
-// eslint-disable-next-line import/no-default-export
-export default {
-  title: 'Components/Tabs',
-  excludeStories: ['HorizontalTabsExample', 'FittedTabsExample', 'VerticalTabsExample', 'StateHookExample'],
-  component: Tabs,
-  subcomponents: {TabList, Tab, TabPanels, TabPanel},
-};
-
-export const HorizontalTabs = (): React.ReactNode => {
-  return <HorizontalTabsExample />;
+// @ts-expect-error story
```

Comment:
Not sure whats better here. Having the `@ts-expect-error` or having the extra code to exclude the `...Example` stories. Thoughts?

Reply:
I think I prefer TS hints to say "this is fine" over adding more code, but I don't really have a strong opinion.

Reply:
I don't really have a strong opinion on this either. We need a tie-breaker @SiTaggart.

Reply:
This may not be a true tie-breaker.

In other stories we type the functional component to return `React.ReactNode`, which is kinda icky but gets around the type error.

In other places like tests I'm pretty partial to not having to do really extreme typescript things it's being picky about because they're tests but, I'm not sure if that's "artisan" and won't bite us in the bum. In those instances where we explicitly expect the type error, I always forget `@ts-expect-error` and I think it's probably the best way to go.

Finally, amusing the `.story` annotations are deprecated anyway https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#hoisted-csf-annotations

Reply:
Lol. I guess this is fine then? We could also create our own re-useable type in the future:
```js
type Story<P> = React.FC<P> & {
  story: {}
  // In the future, the other ones they added to replace story
}
```